### PR TITLE
Add Android support

### PR DESCRIPTION
**Pull request by camertron**

### CODE REVIEW

File: spec/fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - meta_key: leading_space
+      key: " I have a leading space"
+    - meta_key: internal_quotes
+      key: I have "internal" quotes
+    - meta_key: apostrophes
+      key: That's the ticket
+    - meta_key: apostrophes_in_quotes
+      key: That's the ticket
+    - meta_key: escaped_apostrophes_in_quotes
+      key: That's the ticket
```
Comment (jdoconnor) [inline spec/fixtures.yml:46]: copy paste error?

File: spec/xml-extractor_spec.rb [hunk 2]
```diff
@@ -21,7 +21,6 @@
             expected_phrase = expected_phrases[expected_phrase_index]
             expect(expected_phrase).to_not be_nil
             expect(expected_phrase['key']).to eq(actual_phrase.key)
-            # expect(expected_phrase['line_number']).to eq (line_number)
```
Comment (jdoconnor) [inline spec/xml-extractor_spec.rb:24]: why leave it in? Is it only temporarily failing?

**Comment by jdoconnor** [comment]
a couple of comments, then +:neckbeard:

File: spec/fixtures.yml (hunk 1 above)
Comment (camertron) [inline spec/fixtures.yml:46]: All three of these are supposed to be "That's the ticket" :)

File: spec/xml-extractor_spec.rb (hunk 2 above)
Comment (camertron) [inline spec/xml-extractor_spec.rb:24]: Unfortunately Nokogiri doesn't report line numbers correctly under JRuby, so I removed the check from the tests.

File: spec/fixtures.yml (hunk 1 above)
Comment (jdoconnor) [inline spec/fixtures.yml:46]: got it, when i read the description of it being escaped, i was thrown off.

File: spec/xml-extractor_spec.rb (hunk 2 above)
Comment (jdoconnor) [inline spec/xml-extractor_spec.rb:24]: maybe add it as a pending or include a comment on why you're leaving it commented out.

Comment (camertron) [inline spec/xml-extractor_spec.rb:24]: Oh actually I removed it.

**Comment by camertron** [comment]
Sweet thanks!

**Pull request merged**